torchtune/modules/transformer.py

```diff
@@ -96,7 +99,11 @@ def forward(
         # Input tensor and attention output have the same shape
         # [b, s, d]
         # Norm applied before self-attention
-        attn_out = self.attn(self.sa_norm(x), mask=mask, input_pos=input_pos)
+        attn_out = self.attn(
+            self.sa_norm(x),
```

Comment:
This is not using the new attention, right? I guess we would have to merge/solve conflicts